applications/launchpad/gui-react/src/components/NodeBox/NodeBox.test.tsx

```diff
@@ -2,6 +2,7 @@ import { render, screen } from '@testing-library/react'
 import { ThemeProvider } from 'styled-components'
 
 import themes from '../../styles/themes'
+import lightTheme from '../../styles/themes/light'
```

Comment:
you can already access that through themes.light, why import it a second time instead of just
`const lightTheme = themes.light` or using `themes.light` in the test?